source/includes/steps-starting-vsce-connect-with-advanced-connection-settings.yaml

```diff
@@ -246,6 +267,50 @@ content: |
                     For example, the username "X509User" must be 
                     provided as "CN=X509User".
 
+             - id: ldap
+               name: LDAP
+               content: |
+
+                 Select :guilabel:`LDAP` if the deployment
+                 uses :manual:`LDAP </core/security-ldap-external/>`
+                 as its authentication mechanism. If selected, you
+                 must provide the :guilabel:`Username` and
+                 :guilabel:`Password` to authenticate
+                 the user.
+
+             - id: AWS-IAM
+               name: AWS IAM
+               content: |
+
+                 Select AWS IAM if the deployment uses 
+                 `AWS IAM <https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html>`__ 
+                 as its authentication mechanism.
+
+                 .. list-table::
+                    :header-rows: 1
+                    :widths: 40 20 40
+
+                    * - Field
+                      - Necessity
+                      - Description
+
+                    * - AWS Access Key Id
+                      - Required
+                      - An access key ID is a unique identifier 
+                        associated with your AWS account.
```

Comment:
Can we shorten this to:

"Unique identifier associated with your AWS account." ?

Suggest making similar edits to the other AWS field descriptions.